website_v2/gcloud-powershell-website-v2/www/static/app/app.js

```diff
@@ -1,11 +1,11 @@
 (function() {
-    /** This is the angular module for the website. **/
+    /* This is the angular module for the website. */
     var app = angular.module('powershellSite', ['ngRoute']);
 
-    /**
```

Comment:
Ah, I just assumed not to since the style guide I read implied ** was for only the JSDoc comments